bundles/org.openhab.binding.vesync/src/main/resources/OH-INF/update/air-humidifier-instructions.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<update:update-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:update="https://openhab.org/schemas/update-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/update-description/v1.0.0 https://openhab.org/schemas/update-description-1.0.0.xsd">
+
+	<thing-type uid="vesync:airHumidifier">
+		<instruction-set targetVersion="1">
+			<add-channel id="schedulesCount">
+				<type>vesync:deviceAFConfigAutoScheduleCountType</type>
+			</add-channel>
+			<add-channel id="timerExpiry">
+				<type>vesync:deviceAFTimerExpiry</type>
+			</add-channel>
+			<update-channel id="mistLevel">
+				<type>vesync:deviceMistLevelType</type>
+			</update-channel>
+			<update-channel id="warmLevel">
+				<type>vesync:warmLevel</type>
+			</update-channel>
```

Comment:
Only setting the unitHint would not require update channel instructions if im not mistaken.

Reply:
Yeah I was trying to do that but realised it must use those when the instances are made. The only one kept is the fan mode one, as there are new options it does appear to apply with this update-channel added.

Reply:
Not sure if that is actually needed. But it won't harm either. I don;t have the time to test now. You could look at thje json db before and after the new binding is used without performing the upgrade. 
In the json db you can check if the options are cached and or if they are updated.

Reply:
Will leave in just in case. The xml definitions read like it isn't required, but it seemed to make a difference so will leave in just in case.